aws-rds-dbparametergroup/src/main/java/software/amazon/rds/dbparametergroup/ReadHandler.java

```diff
@@ -44,7 +47,11 @@ protected ProgressEvent<ResourceModel, CallbackContext> handleRequest(
                     try {
                         final DBParameterGroup dBParameterGroup = describeResponse.dbParameterGroups().stream().findFirst().get();
                         context.setDbParameterGroupArn(dBParameterGroup.dbParameterGroupArn());
-                        return ProgressEvent.progress(Translator.translateFromDBParameterGroup(dBParameterGroup), context);
+                        final ResourceModel currentModel = Translator.translateFromDBParameterGroup(dBParameterGroup);
+                        if (model.getParameters() != null) {
```

Comment:
Combining this line with https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-rds/pull/406/files#diff-271db357c6f3b9c5912af25ea427a66424f11ba56050a7f16099dab07166cdecR64 
Does describe response returns parameters? In which case the parameters would be missing?